### PR TITLE
Dckesler/page ref

### PR DESCRIPTION
**Pull request by dckesler**

### CODE REVIEW

File: src/plugins/aliases.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const path = require('path');
```
Comment (dckesler) [inline src/plugins/aliases.ts:1]: This plugin allows us to add in webpack aliases. In this case I'm adding `@components/` that is an alias for `src/components/` and can be referenced in our mdx files.

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (dckesler) [inline tsconfig.json:1]: Stuff so we can write our components in typescript.

File: yarn.lock [hunk 3]
```diff
@@ -1238,6 +1238,11 @@
     url-loader "^4.1.1"
     webpack "^5.28.0"
 
+"@docusaurus/module-type-aliases@^2.0.0-beta.0":
```
Comment (dckesler) [inline yarn.lock:1241]: Typescript dev dependencies.

File: src/components/PageRef.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (dckesler) [inline src/components/PageRef.tsx:1]: This is the actual page ref component. It's pretty simple.

File: sidebars.js [hunk 5]
```diff
@@ -81,9 +81,5 @@ module.exports = {
         'celo-holder-guide/celo-recovery',
       ]
     },
-    {
```
Comment (dckesler) [inline sidebars.js:84]: I just now noticed this wasn't a Celo doc it was a docusaurus thing.

File: docusaurus.config.js [hunk 6]
```diff
@@ -85,4 +88,7 @@ module.exports = {
       },
     ],
   ],
+  plugins: [
```
Comment (dckesler) [inline docusaurus.config.js:91]: Here's where I add the aliases plugin.

File: package.json [hunk 7]
```diff
@@ -32,5 +32,13 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```
Comment (dckesler) [inline package.json:36]: This is typescript stuff

**Review by critesjosh — APPROVED** [review]
looks good!

**Pull request merged**